src/main/java/org/fogbowcloud/saps/engine/utils/retry/catalog/GetUser.java

```diff
@@ -16,10 +15,10 @@ public GetUser(Catalog imageStore, String userEmail) {
 	}
 
 	@Override
-	public SapsUser run() throws SQLException {
+	public SapsUser run(){
 		try {
-			return imageStore.getUser(userEmail);
-		} catch (SQLException e) {
+			return imageStore.getUserByEmail(userEmail);
+		} catch (CatalogException e) {
```

Comment:
you should (almost) never do this (catch and do nothing). At least, log it

Reply:
Done